demos/TicTacToe/Classes/MainMenuScene.cpp

```diff
@@ -16,6 +16,8 @@
 
 #include <regex>
 
+#include "cocos2d.h"
```

Comment:
The cocoas2d.h include is already in the header, so it's not needed here, too.

Reply:
Uh, unless you abide my other comment in the header, then it should remain here.  You'll see.  No spoilers!

Reply:
Going to follow up with another pull request to clean up the import/using statements